docs/advanced-stream.md

```diff
@@ -19,13 +19,6 @@ Join the MakeCode developers as we stream MakeCode Arcade live on mixer.com/make
         "youTubePlaylistId": "PLMMBk9hE-SeoHWvTpdezyL9JfF0tocJUT",
         "imageUrl": "/static/advanced-stream/zRewjteisLw.jpg"
     },
-    {
-        "name": "Character Animations",
-        "description": "Join Richard, Shannon, Joey, and Daryl as they add walking and sword swinging animations to an adventure game hero.",
```

Comment:
Deleted?

Reply:
No, this entry is identical to the one above it.